build_parameters.json

```diff
@@ -184,16 +186,30 @@
 							"DESTINATION":     "step/controller:${BRANCH}-${ENVIRONMENT}-java-17",
 							"DESTINATION_PRD": "step/controller:${VERSION}-java-17",
 							"CONFIG": "controller.dockerfile",
-							"IMAGE_BASENAME": "step-controller-${VERSION}-java-17",
+							"IMAGE_BASENAME": "step-controller-${BRANCH}-${ENVIRONMENT}-java-17",
+							"RAW_DIRECTORY": "/step/${SIMPLIFIED_BRANCH_NAME}/sbom/",
 							"SHARE_ON_FTP": false,
 							"DOCKER_BUILD_ARGS": "--build-arg BASE_IMAGE=debian:12-slim --target java-17"
 						},
 						{
+							"DEACTIVATED": "PRODUCTION",
+							"ORIGIN": "step-distribution-controller/target/step-controller-${VERSION}.zip",
+							"DESTINATION":     "step/controller:${BRANCH}-${ENVIRONMENT}-java-21",
+							"DESTINATION_PRD": "step/controller:${VERSION}-java-21",
+							"CONFIG": "controller.dockerfile",
+							"IMAGE_BASENAME": "step-controller-${BRANCH}-${ENVIRONMENT}-java-21",
+							"RAW_DIRECTORY": "/step/${SIMPLIFIED_BRANCH_NAME}/sbom/",
+							"SHARE_ON_FTP": false,
+							"DOCKER_BUILD_ARGS": "--build-arg BASE_IMAGE=debian:12-slim --target java-21"
+						},
+						{
+							"DEACTIVATED": "DEVELOPMENT,INTEGRATION",
```

Comment:
Same question as above, except the other way round: If this is only activated for production, does it make sense to only keep DESTINATION_PRD and to remove DESTINATION?